### ISSUE TITLE
SSAS Tabular 1400: Unhandled exception on running Sync Descriptions

### ISSUE THREAD

**Issue by posthistoric** [BIDeveloperExtensions/bideveloperextensions#20]
![image](https://user-images.githubusercontent.com/1765732/34617243-c60398d2-f208-11e7-9576-7ab6da284d49.png)

**Comment by posthistoric** [comment]
Please let me know if you need any details about our model to repro and fix this.

**Comment by furmangg** [comment]
Yes. What data source is the model? Is it the legacy data source or the modern get data experience?

**Comment by posthistoric** [comment]
Data source is SQL Server 2017. Using the modern Get Data experience.

Thanks,
Vitaly

**Comment by furmangg** [comment]
Off the top of my head I'm 95% sure we don't yet support modern get data. I will leave this open so we can fix the docs and as a reminder to get this working.

**Comment by vitaly-livshits** [comment]
I see. Do you have an ETA on this feature working with Get Data?

**Comment by furmangg** [comment]
No ETA. We haven't even prototyped to see if it's possible. I will leave this open and provide updates as we make progress

**Comment by vitaly-livshits** [comment]
Ok. Perhaps this feature can be updated to return a "Not supported on Get Data" type of error message in the meantime.

**Comment by furmangg** [comment]
The latest 2.3.0 release does return a "BI Developer Extensions does not yet support modern (Power Query) data sources." now. To be perfectly honest, we would love to add support for these types of data sources but it has been such a challenge just keeping BI Developer Extensions compatible with the frequent SSDT updates that we haven't had time for new feature development. I'll leave this issue open so we can provide an update once support for these type of data sources is added.
